tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -7,16 +7,10 @@ import (
 	dto "github.com/prometheus/client_model/go"
 )
 
-// excludedMetrics defines the metrics to ignore.
-// open bug: https://bugzilla.redhat.com/show_bug.cgi?id=2219763
-// Do not add metrics to this list!
-var excludedMetrics = map[string]struct{}{
-	"kubevirt_ssp_operator_up_total":           {},
-	"kubevirt_ssp_template_validator_up_total": {},
-	"ssp_operator_reconciling_properly":        {},
-	"total_rejected_vms":                       {},
-	"total_restored_common_templates":          {},
-}
+// This should be used only for very rare cases where the naming conventions that are explained in the best practices:
+// https://sdk.operatorframework.io/docs/best-practices/observability-best-practices/#metrics-guidelines
+// should be ignored.
+var excludedMetrics = map[string]struct{}{}
```

Comment:
Drop this entirely?

Reply:
We kept it also in other operators, to have a unified way for adding special named metrics. I'd like to keep it if possible.

Reply:
We can keep it but I would not expect any metrics that are non-confirming to be added anymore?